prisma/schema.prisma

```diff
@@ -73,27 +75,28 @@ model SocialMedia {
 }
 
 model Post {
-  id          Int         @default(autoincrement()) @id
-  title       String
-  body        String
-  bodySrc     String      @default("")
-  excerpt     String
-  status      PostStatus  @default(DRAFT)
-  images      Image[]
-  createdAt   DateTime    @default(now())
-  updatedAt   DateTime    @updatedAt
-  publishedAt DateTime?
-  author      User        @relation(fields: [authorId], references: [id])
-  authorId    Int
-  readTime    Int         @default(1)
-  likes       PostLike[]
-  topic       Topic[]     @relation(references: [id])
-  language    Language    @relation(fields: [languageId], references: [id])
-  languageId  Int
-  longitude   Float?
-  latitude    Float?
-  threads     Thread[]
-  postTopics  PostTopic[]
+  id           Int           @default(autoincrement()) @id
+  title        String
+  body         String
+  bodySrc      String        @default("")
+  excerpt      String
+  status       PostStatus    @default(DRAFT)
+  images       Image[]
+  createdAt    DateTime      @default(now())
+  updatedAt    DateTime      @updatedAt
+  publishedAt  DateTime?
+  author       User          @relation(fields: [authorId], references: [id])
+  authorId     Int
+  readTime     Int           @default(1)
+  likes        PostLike[]
+  topic        Topic[]       @relation(references: [id])
+  language     Language      @relation(fields: [languageId], references: [id])
+  languageId   Int
+  longitude    Float?
+  latitude     Float?
+  threads      Thread[]
+  postTopics   PostTopic[]
+  postComments PostComment[]
```

Comment:
heh, this diff is maybe a good argument for why the cute alignment thing the prisma tooling enforces isn't such a great idea. No worries though.

Reply:
Haha agreed!